test/utils.ts

```diff
@@ -63,6 +70,12 @@ function getArbitrumArtifact(contractName: string) {
   }
 }
 
+// Fetch the artifact from the publish package's artifacts directory.
+function getLocalArtifact(contractName: string) {
+  const artifactsPath = `${__dirname}/../../artifacts/contracts`;
+  return findArtifactFromPath(contractName, artifactsPath);
```

Comment:
This might be a bit inefficient, but we could always add caching later to speed things up.